CMakeLists.txt

```diff
@@ -13,8 +13,8 @@ project(Peloton CXX C)
 include(CTest)
 
 # ---[ Peloton version
-set(PELOTON_TARGET_VERSION "0.0.3" CACHE STRING "Peloton logical version")
```

Comment:
We need to add a release system. There have been 1000+ commits since `0.0.3`. We should either stop versioning, or bump it periodically.